packages/babel-plugin-remove-graphql-queries/src/index.ts

```diff
@@ -169,34 +249,38 @@ function getGraphQLTag(path) {
   }
 }
 
-function isUseStaticQuery(path) {
+function isUseStaticQuery(path: NodePath<CallExpression>): boolean {
   return (
     (path.node.callee.type === `MemberExpression` &&
       path.node.callee.property.name === `useStaticQuery` &&
-      path.get(`callee`).get(`object`).referencesImport(`gatsby`)) ||
-    (path.node.callee.name === `useStaticQuery` &&
-      path.get(`callee`).referencesImport(`gatsby`))
+      (path.get(`callee`).get(`object`) as NodePath).referencesImport(
+        `gatsby`,
+        ``
+      )) ||
+    ((path.node.callee as Identifier).name === `useStaticQuery` &&
+      path.get(`callee`).referencesImport(`gatsby`, ``))
   )
 }
 
-export default function ({ types: t }) {
+export default function ({ types: t }): PluginObj {
   return {
     visitor: {
-      Program(path, state) {
+      Program(path: NodePath<Program>, state: {}): void {
         const nestedJSXVistor = {
-          JSXIdentifier(path2) {
+          JSXIdentifier(path2: NodePath<JSXIdentifier>): void {
             if (
-              [`production`, `test`].includes(process.env.NODE_ENV) &&
+              [`production`, `test`].includes(process.env.NODE_ENV || ``) &&
```

Comment:
Nit: this could be `??` rather than `||`

Reply:
(same for a few lines lower)

Reply:
This line was changed in another pull request and is now outdated.